llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -676,6 +678,8 @@ class InnerLoopVectorizer {
   BasicBlock *AdditionalBypassBlock = nullptr;
 
   VPlan &Plan;
+
+  VPBasicBlock *VectorPHVPBB;
```

Comment:
Follow-up: when skeleton creation moves to VPlan creation, ILV should not depend on VPlan or its VPBB's, as was originally the case.